eng/Signing.props

```diff
@@ -47,18 +47,25 @@
 
   <!-- In build signing and publishing without a join point -->
   <ItemGroup Condition="'$(DotNetBuild)' == 'true'">
-    <Artifact Include="$(ArtifactsPackagesDir)**\*.tar.gz;
-                       $(ArtifactsPackagesDir)**\*.zip;
-                       $(ArtifactsPackagesDir)**\*.deb;
-                       $(ArtifactsPackagesDir)**\*.rpm;
-                       $(ArtifactsPackagesDir)**\*.pkg;
-                       $(ArtifactsPackagesDir)**\*.exe;
-                       $(ArtifactsPackagesDir)**\*.msi"
-              Exclude="$(ArtifactsPackagesDir)**\Symbols.runtime.tar.gz"
-              IsShipping="$([System.String]::Copy('%(RecursiveDir)').StartsWith('Shipping'))">
-      <!-- Exclude wixpack.zip files from checksum generation -->
-      <ChecksumPath Condition="$([System.String]::Copy('%(Filename)%(Extension)').EndsWith('.wixpack.zip')) != 'true'">%(FullPath).sha512</ChecksumPath>
-    </Artifact>
+    <Archives Include="$(ArtifactsPackagesDir)**\*.tar.gz;
+                       $(ArtifactsPackagesDir)**\*.zip"
+              Exclude="$(ArtifactsPackagesDir)**\Symbols.runtime.tar.gz;
+                       $(ArtifactsPackagesDir)**\*.wixpack.zip">
+      <ChecksumPath>%(FullPath).sha512</ChecksumPath>
+    </Archives>
+
+    <WixPacks Include="$(ArtifactsPackagesDir)**\*.wixpack.zip" />
+
+    <Installers Include="$(ArtifactsPackagesDir)**\*.deb;
+                         $(ArtifactsPackagesDir)**\*.rpm;
+                         $(ArtifactsPackagesDir)**\*.pkg;
+                         $(ArtifactsPackagesDir)**\*.exe;
+                         $(ArtifactsPackagesDir)**\*.msi">
+      <ChecksumPath>%(FullPath).sha512</ChecksumPath>
+    </Installers>
+
+    <Artifact Include="@(Archives);@(WixPacks);@(Installers)"
+              IsShipping="$([System.String]::Copy('%(RecursiveDir)').StartsWith('Shipping'))" />
```

Comment:
I think all these line changes can be reverted

Reply:
I felt that splitting these out was a little cleaner, but I can revert it to simplify the change.